bin/test

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
+use Translations\Tests\AbstractTranslationTestCase;
+
+$phpCsFixer = 'vendor/bin/php-cs-fixer';
+$phpunit    = 'vendor/bin/phpunit';
+
+if (CLI::isWindows()) {
+    $phpCsFixer = strtr($phpCsFixer, '/', '\\');
+    $phpunit    = strtr($phpunit, '/', '\\');
+}
+
+CLI::write('Running `composer update`...', 'black', 'green');
+passthru('composer update --ansi');
+CLI::write();
+
```

Comment:
Running `composer update` over and over seems to be unnecessary. Can we just check instead if `vendor/autoload.php` is existing and if not we exit?

Reply:
It is needed to update `codeigniter4/codeigniter4`.

Reply:
This script is for translators to check untranslated items with one command.
If they want to run tests over and over, they can use like `vendor/bin/phpunit --filter JapaneseTranslationTest`.